sql/moz-fx-data-shared-prod/telemetry_derived/desktop_retention_v1/metadata.yaml

```diff
@@ -8,6 +8,10 @@ labels:
   owner1: example
 scheduling:
   dag_name: bqetl_desktop_retention_model
+  date_partition_parameter: metric_date
+  date_partition_offset: -27
+  parameters:
+  - submission_date:DATE:{% raw %}{{ds}}{% endraw %}
```

Comment:
Remove the raw surrounding {{ds}} this is only needed if using a generator

Reply:
ok, thank you.